src/lib/webpack/webpack-base-config.js

```diff
@@ -141,13 +134,6 @@ export default (env) => {
 									loader: 'sass-loader',
 									options: { sourceMap: true }
 								}
-							},
-							{
```

Comment:
What the best way for templates to add these?

Reply:
`preact.config.js` is the only way the user can tweak the webpack config in CLI as of now.

Reply:
Template authors have to define their own dependencies, as a rule. As of now, the template can also include a preact.config.js, but in the near future, I'll add a `meta.js` file (from root) that will have postinstall hooks and behaviors. Hat tip to Vue for that.

Reply:
Perfect